hepcrawl/spiders/oup_spider.py

```diff
@@ -192,12 +197,12 @@ def download_files_from_ftp(self, ftp_folder):
                 file_name = '%s_%s' % (
                     filename_prefix, os.path.basename(file_path))
                 local_filename = os.path.join(self.target_folder, file_name)
-                host.download(file_path, local_filename)
 
+                download_file_from_ftp()
```

Comment:
why this change? You can decorate a method. Also I would recommend be more specific in the Exception

Reply:
It shouldn’t be class or function for a decorator?

![Screen Shot 2022-02-21 at 15 58 18](https://user-images.githubusercontent.com/36658177/154979976-b8def9b3-11b4-4079-a206-b3959316a906.png)